Web/starter_project_iconics/cypress/support/commands.ts

```diff
@@ -34,4 +34,4 @@
 //       visit(originalFn: CommandOriginalFn, url: string, options: Partial<VisitOptions>): Chainable<Element>
 //     }
 //   }
-// }
\ No newline at end of file
+// }
```

Comment:
if these space changes are happening when you have run lint fix, then that is fine, but if you added them another way i recommend not doing that

Reply:
Its a lint fix thing.